src/main/java/org/sonarqube/geekshubs/FizzBuzz.java

```diff
@@ -4,11 +4,18 @@
  * Created by ricardo on 23/02/17.
  */
 public class FizzBuzz {
-  public static String getResult(int number) {
-    if (number % 15 == 0) return "fizzbuzz";
-    else if (number % 3 == 0) return "fizz";
-    else if (number % 5 == 0) return "buzz";
+
+  public String getResult(int number) {
+    if (number % 15 == 0) {
+      return "fizzbuzz";
+    } else if (number % 3 == 0) {
+      return "fizz";
+    } else if (number % 5 == 0) {
+      return "buzz";
+    }
+    
+    System.out.println(Integer.toString(number));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/sonarqubegeekshubs/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarqube.geekshubs%3Afizzbuzz&pullRequest=1&issues=AWQJfrZ4uoX1TGVkROIM&open=AWQJfrZ4uoX1TGVkROIM)

[ISSUE_KEY]: AWQJfrZ4uoX1TGVkROIM